ci/citest/acceptance-test/multibox/ind-steps/step-vdc-images/init.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+for img in "${IMAGES[@]}" ; do
+    item=$(grep "$img" <<< "$imgIndex")
+    IFS=';' read -a parts <<< "$item"
+    IFS=';' eval 'folders=($item)'
+    imgRemoteDir="${parts[-1]}"
+
+    prepare_container_image "${img}" "${folders[0]}/${folders[1]}/${folders[2]}" "$imgRemoteDir"
+    add_container_image "${img}" "${folders[0]}/${folders[1]}/${folders[2]}"
+done
+
+(
+    # resync again after images have been downloaded
+    $starting_step "Synching guestroot for ${vm_name}"
```

Comment:
It's a little inefficient to sync twice. I understand how you're feeling though. I've also ran into problems where I wanted to do something before rsync but wasn't able to. Therefore It might be better to rethink the way our steps are set up. I'm thinking something like this.

```
pre_rsync.sh
*call rsync*
post_rsync.sh
pre_boot.sh
*boot vm*
post_boot.sh
```

Doesn't necessarily need to be in this pull request since it's a bigger more all encompassing design change but I'm also a little afraid of postponing such refactors and letting code like this build up. What's your opinion?